helm_deploy/hmpps-book-secure-move-frontend/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
We deploy one artefact per repository, so do we need the `hmpps-book-secure-move-frontend` directory here?

Reply:
It's the folder for the chart and relevant info, while the files outside are used in helm commands.